lib/libxdp/libxdp.map

```diff
@@ -79,4 +79,5 @@ LIBXDP_1.3.0 {
 
 LIBXDP_1.4.0 {
 		xsk_umem__create_with_fd;
+		xsk_umem__create_opts;
```

Comment:
We already released libxdp 1.4.0, so this should go in a new LIBXDP_1.5.0 section.